schemas.py

```diff
@@ -0,0 +1,46 @@
+from datetime import date
+from typing import List
+
+from pydantic import BaseModel
+
+
+class AuthorBase(BaseModel):
+    name: str
+    bio: str
+
+
+class AuthorCreate(AuthorBase):
+    pass
+
+
+class AuthorList(AuthorBase):
+    id: int
+    books: List["Book"]

```

Comment:
You're referencing the Book model within the books field using a forward reference (List["Book"]). However, the Book class is defined after the AuthorList class, so Python hasn't seen the Book class when it first encounters the forward reference.